test/fixtures/v1-co-81657-state-utility-lowincome.json

```diff
@@ -385,7 +385,8 @@
       "program": "Rebates & Incentives",
       "program_url": "https://www.walkingmountains.org/sustainability-hub/energy-efficiency-programs/rebates-incentives/",
       "item": {
-        "type": "smart_thermostat"
+        "type": "smart_thermostat",
+        "name": "Smart Thermostat"
```

Comment:
I'm not sure why Colorado got updated as part of this PR. Happy to ditch changes to this file if I made some mistake here.

Reply:
Will come by for a full review later today, but wanted to say that this is unrelated and will be fixed with https://github.com/rewiringamerica/api.rewiringamerica.org/pull/320. You can revert this file.